game/src/test/java/action/DamageActionTest.java

```diff
@@ -0,0 +1,19 @@
+package action;
+
+import object.Damageable;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public class DamageActionTest {
+
+    @Test
+    public void shouldCallTakeDamageWhenDoAction() {
+        Damageable damageable = Mockito.mock(Damageable.class);
+        int expectedDamage = 1;
```

Comment:
lets inline it. We mostly expect that "takeDamage()" method is invoked not what damage has been dealt